include/z64effect.h

```diff
@@ -257,7 +257,6 @@ typedef struct {
     /* 0x18 */ u8 unk18; // Always 1?
 } EffectSsOverlay; // size = 0x1C
 
-//! TODO: Review reuse of vec/gfx/actor fields across all effects
```

Comment:
Reviewed these:
vec: used to store vector information of different kinds like position and rotation
gfx: used to store dls and textures, and in 1 instance stores a collisionpoly (new to MM)
actor: used to store actors, and in 1 instance stores an s16* (usually the actors access struct members directly with no opportunity to cast to a temp, so a void* is unlikely.)

Reply:
Right, so we're happy that the srchball instance should stay as a bug? Do we want to add a warning for it?

Reply:
Personally I would prefer just the bug tag.

Reply:
I'd like to know what others think, I think I'd prefer to have the warning myself, but I know @AngheloAlf prefers not, at least for now.